pytensor/graph/basic.py

```diff
@@ -558,6 +558,46 @@ def get_parents(self):
             return [self.owner]
         return []
 
+    def convert_string_keys_to_pytensor_variables(self, inputs_to_values):
+        r"""Convert the string keys to corresponding `Variable` with nearest name.
+
+        Parameters
+        ----------
+        inputs_to_values :
+            A dictionary mapping PyTensor `Variable`\s to values.
+
+        Examples
+        --------
+
+        >>> import numpy as np
+        >>> import pytensor.tensor as at
+        >>> x = at.dscalar('x')
+        >>> y = at.dscalar('y')
+        >>> z = x + y
+        >>> np.allclose(z.eval({'x' : 3, 'y' : 1}), 4)
+        True
```

Comment:
This does not show a use of the function. In any case we don't need a docstring with an example I think

Reply:
Since we are making it internal to eval I also think that the doc-string will be no more required.